File: java/iniciante/ex-1008.java
package application;

import java.util.Locale;
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int nFuncionario, nHoras;
		double salarioHora, salario;
		
		nFuncionario = sc.nextInt();
		nHoras = sc.nextInt();
		salarioHora = sc.nextDouble();
		
		salario = nHoras * salarioHora;
		
		System.out.println("NUMBER = " + nFuncionario);
		System.out.printf("SALARY = U$ %.2f", salario);
		System.out.println();
		
		sc.close();

	}

}

File: c++/iniciante/ex1008.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main(){
	
	int nFuncionario, nHoras; 
	double salarioHora, salario;
	
    cin >> nFuncionario;
    cin >> nHoras;
    cin >> salarioHora;
    
    salario = nHoras*salarioHora;
    
    cout << "NUMBER = " << nFuncionario << endl;
    cout << "SALARY = U$ " << fixed << setprecision(2) << salario << endl;
    
    system("pause");
    
    return 0;
}
